airflow/cli/cli_parser.py

```diff
@@ -626,6 +626,16 @@ def positive_int(value):
     ),
     action="store_true",
     default=False)
+ARG_CONN_EXPORT_FORMAT = Arg(
+    ('--format',),
+    help='Output format of the file for exporting the connections',
+    default='json',
+    type=str)
```

Comment:
Can you consider to use choices?
https://docs.python.org/3/library/argparse.html#choices